### ISSUE TITLE
[Visual requirement - Make code - Share project]: Luminosity ratio of the "Getting Started" text does not meet minimum contrast ratio 4.5:1

### ISSUE THREAD

**Issue by Yamini234u** [microsoft/pxt-microbit#5469]
### **User Experience**
Low vision users will get impacted as they will not be able to view the static text on the page properly & will face difficulties while reading the content on the page if the luminosity ratio of the Getting Started" text does not meet the minimum requirement of 4.5:1 ratio.

Note:
User credentials should NOT be included in the bug.

### **Repro Steps:**

1. Launch application URL: https://arcade.makecode.com/github
2. "GitHub with MakeCode" page will open
3. Run AI 
4. Observe and verify whether "Getting Started" text meets minimum contrast ratio or not.

### **Actual Result:**
 Luminosity ratio of the "Getting Started" text does not meet minimum contrast ratio 4.5:1.

Observation: The actual luminosity ratio of the text is 3.01.

### **Expected result:**
Luminosity ratio of the "Getting Started" text should meet minimum contrast ratio 4.5:1.

###  **Issue**
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - [](https://accessibilityinsights.io/info-examples/web/color-contrast)
Target application: GitHub with MakeCode - https://arcade.makecode.com/github
### **Element path:**
.primary > p
### **Snippet:**
`<p>Getting started</p>`

### **How to fix:**
Fix any of the following:
 Element has insufficient color contrast of 3.01 (foreground color: #ffffff, background color: #f76820, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
### **MAS Reference:**
[MAS 1.4.3 – Contrast (Minimum)](https://aka.ms/MAS1.4.3)
**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA
### 
 **DOM details available:**
Yes(Please refer to the attachment)
### **Reference Links:**
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDc3NDY0MzExZDkxNTRlYmRhNzk2YTExYzE5Y2I2N2ZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3ODcwNDg4MTQ1NjM2NzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=VHBCSFBzMTdmWExVNTk3MzZlTHdiME4rcFo3d3lyYk5kTTIvRjhLdTN3QT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-chbhardwaj%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
### **Test Environment:**
OS Version: Windows 11 23H2 (OS Build:22631.3007)
Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
Test Matrix: Chrome + JAWS
Tool: Color Contrast Analyser
URL: https://makecode.microbit.org/.
<img width="1185" alt="MAS1 4 3-Luminosity ratio of the Getting Started text does not meet minimum contrast ratio" src="https://github.com/microsoft/pxt-microbit/assets/117158108/08b7bb75-dd5d-4cee-aaa3-bb4e217a6cab">

**Comment by Yamini234u** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#A11yAuto;#K4W;#HCL;#E2E-HCL-MakeCode-Jan2024;#Benchmark;#Win11-Chrome;#MAS1.4.3

**Comment by vroopea** [comment]
As checked the issue on URL: https://makecode.microbit.org/. The issue is still repro and severity is fine. #Regressed:04-08-24;InternalReviewed-Apr2024
